docs/src/app/components/pages/components/IconButton/ExampleComplex.jsx

```diff
@@ -1,19 +1,25 @@
 import React from 'react';
 import FontIcon from 'material-ui/lib/font-icon';
 import IconButton from 'material-ui/lib/icon-button';
+import ActionHome from 'material-ui/lib/svg-icons/action/schedule';
```

Comment:
This is not `ActionHome`.
What about using the same icon for the three different type?
This will help to make sure that they are treated equally.